lib/common/io.ts

```diff
@@ -63,3 +63,95 @@ export enum SocketDomain {
     /** UNIX domain socket/Windows named pipes */
     LOCAL = 2,
 }
+
+/**
+ * The amount of detail that will be logged
+ * @category Logging
+ */
+export enum LogLevel {
+    /** No logging whatsoever. */
+    NONE = 0,
+    /** Only fatals. In practice, this will not do much, as the process will log and then crash (intentionally) if a fatal condition occurs */
+    FATAL = 1,
+    /** Only errors */
+    ERROR = 2,
+    /** Only warnings and errors */
```

Comment:
Extremely Trivial: This is the only comment that explicitly calls out itself along with another log category. It's unlikely but this may cause someone to assume that later levels are not inclusive of previous log levels.